Various improvements

zoontek opened the pull request:
## What it does

- Add a valid SPDX license expression to `package.json` + fix the `LICENSE` file
- Format all files using `prettier`
- Add `lint-staged` to format all staged files
- Add `type input / output` to inputs and outputs to format and highlight TS correctly
- Switch to `jsonc` (https://github.com/github/linguist/issues/4153)
- Fix some typos

ChucKN0risK commented on README.md:
```diff
@@ -1,21 +1,19 @@
 
 ## What is Specify?
-Specify helps you unify your brand identity by collecting, storing and distributing design tokens and assets—automatically. 
+
```
Please remove line jump

zoontek replied:
@ChucKN0risK You cannot with these prettier settings (it actually render the same as it's markdown)

ChucKN0risK commented on README.md:
```diff
@@ -1,21 +1,19 @@
 ## What is Specify?
-Specify helps you unify your brand identity by collecting, storing and distributing design tokens and assets—automatically. 
+
+Specify helps you unify your brand identity by collecting, storing and distributing design tokens and assets—automatically.
```
Specify helps you unify your brand identity by collecting, storing and distributing design tokens and assets — automatically.

Specify is a design API facilitating the distribution of your design data within your design system.

![design-api-full-dark](https://user-images.githubusercontent.com/3913276/156588899-4a8e0183-f81e-40b2-98e9-ad518d8b6813.jpg)

ChucKN0risK commented on README.md:
```diff
@@ -38,6 +36,7 @@ Let's say you want to create a parser named `my-parser`.
 ### Creating a valid parser
 
 To be valid, your parser needs:
+
```
Please remove line jump

ChucKN0risK commented on README.md:
```diff
@@ -49,11 +48,13 @@ To be valid, your parser needs:
 ## Usable libraries
 
 For now, our parsers only use the following libraries:
+
```
Please remove line jump

ChucKN0risK commented on README.md:
```diff
@@ -49,11 +48,13 @@ To be valid, your parser needs:
 - [svgo](https://github.com/svg/svgo/)
 
 If you need another library to develop your parser:
+
```
Please remove line jump

ChucKN0risK commented on README.md:
```diff
@@ -62,7 +63,8 @@ If you need another library to develop your parser:
 
-To use our design tokens seed: 
+To use our design tokens seed:
+
```
Please remove line jump

ChucKN0risK commented on README.md:
```diff
@@ -49,11 +48,13 @@ To be valid, your parser needs:
 - [lodash](https://github.com/lodash/lodash)
 - [svgo](https://github.com/svg/svgo/)
 
 If you need another library to develop your parser:
```
How to add another library to develop your parser:

zoontek commented on .github/workflows/pull_request_template.md:
```diff
@@ -1,6 +1,5 @@
+<!-- Quick description -->
 
-## Additionnal informations
+## Additional informations
```
@ChucKN0risK I guess there should be more than one information. It might be a list, no?

zoontek commented:
@ChucKN0risK @AMoreaux I answered to remarks

AMoreaux commented:
Thanks a lot for your contribution @zoontek 🎊 I just create a new PR #117  with your commits renamed to fit with our guidelines. Moreover, I omit changes on the license that require more discussion. Fill free to comment it 🙂

zoontek commented:
@AMoreaux I just added a commit to fix code of conduct links (https://github.com/Specifyapp/parsers/pull/116/commits/7af44a6505effbc686831c3e50fd1dbe1a95cd10).

It's exactly the same licence, I just added a license property in `package.json` in a format that npm could understand (it must respect https://spdx.org/licenses) - because currently it says **"License: none"** on npm (https://www.npmjs.com/package/@specifyapp/parsers)

But also in a format that GitHub could understand (`LICENSE` file, not `LICENCE` which is the french word 😛) and not in markdown so it could display license details.

Example

<img width="1259" alt="Screenshot 2022-03-03 at 19 42 03" src="https://user-images.githubusercontent.com/1902323/156631091-2db0793c-7738-4a2b-98e1-4853a95ea8e0.png">

AMoreaux commented:
Close in favor of #117
